5128 TOGGLE behandlingsmeny juster tilgjengelige valg

fredrik-oh opened the pull request:
https://jira.adeo.no/browse/MELOSYS-5128
https://confluence.adeo.no/pages/viewpage.action?pageId=459902674

Tilpasser visning av menypunkter under behandlingsmenyen for toggle behandleAlleSaker.  Implementert confluence tabellen kun for eksisterende menypunkt:

- Avslå søknad pga. manglende opplysninger
- Behandlingen er henlagt/trukket
- Behandlingen er bortfalt (het tidligere "Kan ikke behandles i Melosys")

De nye menypunktene skal implementere tabellen i egne jiraer

Commit:
Justerte logikk for visning av menypunkter i behandlingsmeny. La endringene bak behandleAlleSakerToggle

Commit:
Forenklet skalViseBehandlingenErBortfalt

Commit:
Oppdaterte avsluttsak.test.tsx

martonSkjevelandNav reviewed (approved):
Logikken ser riktig ut ut ifra tabell.
Eneste jeg vil pirke litt på er egen preferanse av `behandleAlleSakerToggle === "disabled"` over `behandleAlleSakerToggle !== "enabled"`

janitamarielle commented on src/felleskomponenter/informasjonlinje/behandlingsmeny/avsluttsak.tsx:
```diff
@@ -78,54 +81,122 @@ const AvsluttSak = ({
     }
+    if (sakstype === TRYGDEAVTALE) {
+      return (
+        ![HENVENDELSE, KLAGE].includes(behandlingstype) &&
```
Er det ikke litt rart at denne er omvendt av de andre sakstypene?

IsaSalemHame commented on src/felleskomponenter/informasjonlinje/behandlingsmeny/avsluttsak.tsx:
```diff
@@ -78,54 +81,122 @@ const AvsluttSak = ({
+
+    if (sakstype === EU_EOS) {
+      return (
+        [FØRSTEGANG, NY_VURDERING].includes(behandlingstype) &&
+        [
+          UTSENDT_ARBEIDSTAKER,
+          UTSENDT_SELVSTENDIG,
+          ARBEID_TJENESTEPERSON_ELLER_FLY,
+          ARBEID_FLERE_LAND,
+          IKKE_YRKESAKTIV,
+          ARBEID_KUN_NORGE,
+          PENSJONIST,
+        ].includes(behandlingstema)
```
begynner å bli en laang lang funksjon. Vi kan ikke ha litt av denne logikken i egne metoder som beskriver litt hva denne listen er?

IsaSalemHame replied:
Kanskj eikke ha egen metode, men gi ha de listene som const med et beskrivende navn..?

Commit:
Fikset merge conflict og avsluttsak.test.tsx etter merge, flere tilgjengelige menyvalg i de fleste test cases

Merge branch 'master' into MELOSYS-5128_Behandlingsmeny_juster_tilgjengelige_valg

# Conflicts:
#	src/felleskomponenter/informasjonlinje/behandlingsmeny/avsluttsak.test.tsx
#	src/felleskomponenter/informasjonlinje/behandlingsmeny/avsluttsak.tsx
